src/main/java/pt/iscte/es1/antiSpamFilter/infrastructure/NsgaRToEpsCompiler.java

```diff
@@ -1,11 +1,45 @@
 package pt.iscte.es1.antiSpamFilter.infrastructure;
 
-public class NsgaRToEpsCompiler {
+import javafx.scene.control.Alert;
+import pt.iscte.es1.antiSpamFilter.AntiSpamFilterConstants;
+import pt.iscte.es1.antiSpamFilter.gui.AlertMessage;
+
+import java.awt.Desktop;
+import java.io.File;
+import java.io.IOException;
 
-	public static final String NGSAII_R_FILE = "HV.Boxplot.R";
-	public static final String NGSAII_EPS_FILE = "HV.Boxplot.eps";
+/**
+ * Compile R to EPS
+ */
+public class NsgaRToEpsCompiler {
 
-	public void compile() {
+	private static final String R = "/R";
+	private static final String NSGAII_R_FILE = "/HV.Boxplot.R";
+	private static final String NSGAII_EPS_FILE = "/HV.Boxplot.eps";
 
+	/**
+	 * Converts the R file into EPS file and presents it to the User through the Default Application
+	 * that is specified from the user to execute/open EPS files.
+	 */
+	public void compileAndShow() {
+		Runtime rt = Runtime.getRuntime();
+		Process process = null;
+		try {
+			process = rt.exec("Rscript " + NSGAII_R_FILE, null,
+				new File(AntiSpamFilterConstants.ANTI_SPAM_STUDY + R + NSGAII_R_FILE));
+			if (process != null) {
+				Desktop.getDesktop().open(
+					new File(AntiSpamFilterConstants.ANTI_SPAM_STUDY + R + NSGAII_EPS_FILE));
+			} else {
+				new AlertMessage(Alert.AlertType.WARNING,"Error Opening File",
```

Comment:
Throw an exception instead of alert. Launch alert in the controller. Separation of concerns.